test/Macros/PointerBounds/CountedBy/MutableSpan.swift

```diff
@@ -1,6 +1,6 @@
 // REQUIRES: swift_swift_parser
 // REQUIRES: pointer_bounds
-
+// XFAIL: OS=windows-msvc
```

Comment:
@DougGregor if MutableSpan is unavailable, shouldn't we XFAIL this on all platforms?

Reply:
I suspect that the macro is being expanded but nothing ever type-checks it, so it's "working" for now but will fail with subsequent changes. We can disable then.